examples/flex-flows/chat-stream/README.md

```diff
@@ -0,0 +1,117 @@
+# Chat stream
+A chat flow defined using class entry that return output in stream mode. It demonstrates how to create a chatbot that can remember previous interactions and use the conversation history to generate next message.
+
+## Prerequisites
+
+Install promptflow sdk and other dependencies in this folder:
+```bash
+pip install -r requirements.txt
```

Comment:
maybe add a sample using prompty in flex flow

Reply:
Changed code evaluation sample to prompty + flex flow.